backend/src/main/java/codezap/member/controller/SpringDocMemberController.java

```diff
@@ -26,321 +25,65 @@ public interface SpringDocMemberController {
-                                      "instance": "/signup"
-                                    }
-                                    """
-                            ),
-                            @ExampleObject(name = "사용자명 글자수 오류", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "BAD_REQUEST",
-                                      "status": 400,
-                                      "detail": "사용자명이 입력되지 않았습니다.",
-                                      "instance": "/signup"
-                                    }
-                                    """
-                            ),
-                    }
-            )
-    )
-    @ApiResponse(
-            responseCode = "409",
-            description = "이메일 또는 사용자명 중복",
-            content = @Content(
-                    schema = @Schema(implementation = ProblemDetailSchema.class),
-                    examples = {
-                            @ExampleObject(name = "이메일 중복", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "CONFLICT",
-                                      "status": 409,
-                                      "detail": "이메일이 이미 존재합니다.",
-                                      "instance": "/signup"
-                                    }
-                                    """
-                            ),
-                            @ExampleObject(name = "사용자명 중복", value = """
-                                    {
-                                      "type": "about:blank",
-                                      "title": "CONFLICT",
-                                      "status": 409,
-                                      "detail": "사용자명이 이미 존재합니다.",
-                                      "instance": "/signup"
-                                    }
-                                    """
-                            )
-                    }
-            )
-    )
+    @ApiErrorResponse(status = HttpStatus.BAD_REQUEST, instance = "/signup", errorCases = {
+            @ErrorCase(description = "이메일 입력 없음", exampleMessage = "이메일이 입력되지 않았습니다."),
+            @ErrorCase(description = "이메일 형식 오류", exampleMessage = "이메일 형식이 아닙니다."),
+            @ErrorCase(description = "이메일 글자수 오류", exampleMessage = "이메일은 255자 이하로 입력해주세요."),
+            @ErrorCase(description = "비밀번호 입력 없음", exampleMessage = "비밀번호가 입력되지 않았습니다."),
+            @ErrorCase(description = "비밀번호 형식 오류", exampleMessage = "영어와 숫자를 포함해야합니다."),
+            @ErrorCase(description = "비밀번호 글자수 오류", exampleMessage = "비밀번호는 8~16자 사이로 입력해주세요."),
+            @ErrorCase(description = "사용자명 입력 없음", exampleMessage = "사용자명이 입력되지 않았습니다."),
+            @ErrorCase(description = "사용자명 글자수 오류", exampleMessage = "사용자명이 입력되지 않았습니다.")
```

Comment:
두가지 에러 케이스를 하나의 에러 케이스로 합쳐도 좋을 것 같아요 ~

```suggestion
            @ErrorCase(description = "사용자명 입력이 없거나 글자수 오류", exampleMessage = "사용자명이 입력되지 않았습니다."),
```

Reply:
이번 pr은 기존에 작성된 문서 내용은 수정하지 않았습니다.
기능에 대한 리팩토링이 아니라 이슈로 남겨두고 따로 pr 보낼게요~